idd/Energy+.idd.in

```diff
@@ -48642,6 +48642,64 @@ ZoneHVAC:AirDistributionUnit,
        \type object-list
        \object-list DesignSpecificationAirTerminalSizingName
 
+ZoneHVAC:ExhaustControl,
```

Comment:
Is it worth putting `min-fields` on these new objects?

Reply:
Thanks! I think a `min-fields 4` would be helpful for the AirLoopHVAC:ExhaustSystem one, which I will add to it. The ZoneHVAC:ExhaustControl might benefit from some number as well but with only min-fields numbers it is not fully defined--there would be multiple different ways to meet that min fields number but then but still be insufficient--so for this one I would like to pass.